src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tasks that automate the job of moving projects in and out of symbolicator's low priority queue based
+on symbolication metrics stored in Redis.
+
+This has three major tasks, executed in the following general order:
+1. Scan for new suspect projects in Redis that need to be checked for LPQ eligibility. Triggers 2 and 3.
+2. Determine a project's eligibility for the LPQ based on their recorded metrics.
+3. Remove some specified project from the LPQ.
+"""
+
+import logging
+
+import sentry_sdk
+
+from sentry.processing.realtime_metrics import realtime_metrics_store
+from sentry.processing.realtime_metrics.base import BucketedCount
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.tasks.symbolicator.scan_for_suspect_projects",
+    queue="symbolicator.compute_low_priority_queue",
+    ignore_result=True,
+    soft_time_limit=10,
+)  # type: ignore
+def scan_for_suspect_projects() -> None:
+    _scan_for_suspect_projects()
+
+
+def _scan_for_suspect_projects() -> None:
+    suspect_projects = set()
+    for project_id in realtime_metrics_store.get_lpq_candidates():
+        suspect_projects.add(project_id)
+        compute_lpq_eligibility(project_id)
+
+    current_lpq_projects = realtime_metrics_store.get_lpq_projects() or set()
```

Comment:
ok, now i get why `get_lpq_projects` is needed and it kind of sucks.

the solution would be always create a new set of selected projects in a new redis key, then ask redis for the diff, finally copy the new lpq set to the canonical location.

but.  all that probably doesn't matter and we may as well take the O(n) hit for this to trade off the complexity until O(n) becomes a problem.

Reply:
linking back to the original comment asking why `get_lpq_projects()` even needs to exist for personal reference: https://github.com/getsentry/sentry/pull/28757#discussion_r713830740

looks like you've found out why here, though! what isn't immediately obvious to me is when the right time would be to copy the new LPQ set over. the work of determining project eligibility is done asynchronously in separate tasks, so how do you determine when to copy things over? at the start of the next computation task?

in any case, i think an O(n) hit on a set that's meant to be very small to begin with should be fine, especially since we intend to treat the presence of a project in the LPQ as an incident.